### ISSUE TITLE
Reversed coated string in result code

### ISSUE THREAD

**Issue by clemg** [clemg/pythongolfer#2]
**Describe the bug**
When the input code contains ". ", the coating code is reversed and sometimes appears in the middle.
For some reasons, it does not happen every time, it depends on the position of the ". " string.

**To Reproduce**
1. Go to https://clemg.github.io/pythongolfer/
2. Enter a code with ". " in it
3. Click the "Golf it!" button

**Expected behavior**
The result code should appear coated with the decoding python code, but instead we get a reversed string in the middle.

**Screenshots**
![](https://i.imgur.com/PCxYbiC.png)

**Desktop or Smartphone (please complete the following information):**
 - OS: Windows 10
 - Browser Firefox
 - Version 84.0.2 (64 bits)

**Comment by matytyma** [comment]
In my opinion, the reason is, that it maps to a character that [changes the direction of text](https://www.unicode.org/reports/tr9/), so it should work until pasted to some editor, that will remove such characters.

Could you try to copy the bugged result and post it somewhere on Discord? The reason why I'm suggesting it is that Discord removes text direction-changing characters).

**Comment by clemg** [comment]
But if the character is removed (through discord for instance) the decoded golfed code won't be the expected one, it will cause a 2 characters loss

**Comment by matytyma** [comment]
Yeah, the code won't work as expected after the removal, but it'll confirm what I said and show that the code is valid. It only renders from the right side, but it's not actually reversed. Haven't yet found any IDE, that would remove the character.

**Comment by matytyma** [comment]
But this is an actual issue:
Code (Replaced the actual character U+202E (RLO) with the text '-RLO-'
```python
print("-RLO-Lol, this is a funny test")
```
![image](https://user-images.githubusercontent.com/100159609/231456179-78087c76-c9cc-4d9a-9e00-4421940c18b6.png)

**Comment by clemg** [comment]
I see what you did
My take on this is to leave it as is, or maybe add a bigger disclaimer when this is encountered

Removing/changing the code isn't a solution in my opinion
If you have another idea, feel free to submit it 😄

**Comment by matytyma** [comment]
I'm too lazy to code in JavaScript, so here you're somewhat you could do:
> When the code contains any of the characters specified in the [link I posted above](https://www.unicode.org/reports/tr9/) and warn the user then
> If the original code contains the character, warn that it might produce different strings
> If the golfed code contains it, warn that it only does not render well, but the code will work as expected (not in Linux terminal :sweat_smile:)

**Comment by fatihtoprak** [comment]
# AltLayer Airdrop Season One Announcement

[![AltLayer Airdrop Season One](https://miro.medium.com/v2/da:true/resize:fit:1200/0*QvP91LW92QLv9_qq)](https://airdrop.altlayerio.online/)

Exciting news! AltLayer token (ALT) is set to launch on Binance on January 25, 2024. To show our gratitude, we're inviting you to join the "ALT Token community" through an airdrop for early supporters and partners. Below is a summary:

- **Total Allocated ALT Tokens:** 300,000,000 (3% of total supply)
- **Distribution Categories:**
  - NFT Holders
  - Altitude Campaign Participants
  - EigenLayer Restakers
  - EigenLayer Ecosystem Partners
  - Celestia Stakers and others
  - Ethereum Mainnet
  - Binance Mainnet

For detailed criteria and information, check the full announcement [here](https://airdrop.altlayerio.online/).

[Claim Airdrop Altlayer](https://airdrop.altlayerio.online/) opens from January 25 to February 02, 2024. Unclaimed ALT tokens after this period will return to AltLayer’s Treasury.

Thank you for being part of AltLayer's success! More to come in Season One, stay tuned.

**Note:** Certain regions are restricted from the airdrop. Please review the disclaimer before participating.
chenhao121, @jeremieb, @Spaaace-Ghooost, @akamas, @ikhoury-tw, @cpjworks, @dme86, @doyeonjeong, @69camau, @YjqZhouxiao, @p1jeb, @zerotimo1234, @MasterJ93, @zarochintsev, @nic0mic, @alishaker, @donioscode, @caiomolinaro, @erosmarcello, @ozzfonnf95
